docs/UserGuide.md

```diff
@@ -195,197 +245,166 @@ Examples:
-* `find_session John Lee` returns all John Lee in TutorBuddy
-* `find_session Zach` returns nothing-->
-<!-- END OF COMMENT OUT FOR FIND SESSION -->
+### Managing Sessions
 
 #### Adding a single tuition session: `add_session`
 
-Adds a single tuition session to the TutorBuddy
+Adds a single tuition session to TutorBuddy.
 
 Format: `add_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE`
 
 * `STUDENT_NAME` should match the exact student’s name in TutorBuddy
-* `DATE` should be in YYYY-MM-DD format
-* `TIME` should be in HH:MM 24-hr format
 * `DURATION` should be in minutes
 * `FEE` should be the total tuition fee for the total duration
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+<div markdown="span" class="alert alert-primary">:bulb: Tip:
 TutorBuddy takes care of overlapping session for you by giving a gentle prompt, so you don't have to worry about it.
 </div>
 
-Examples:
+Example:
 * `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80`
 
 #### Adding a recurring tuition session: `add_rec_session`
 
-Adds a recurring tuition session to the TutorBuddy
-
-Format: `add_rec_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE b/INTERVAL e/LASTDATE`
+Adds a recurring tuition session to TutorBuddy that allows you to specify the number of days between each recurring session.
 
-Similar to arguments for adding a single tuition session, except:
-* `DATE` refers to the starting date of the session
+Format: `add_rec_session n/STUDENT_NAME d/START_DATE e/END_DATE b/INTERVAL t/TIME k/DURATION s/SUBJECT f/FEE `
 
-Arguments additional to adding a single tuition session:
-* `INTERVAL` should be in days, including the date from and excluding the date to <br/>
+Arguments similar to `add_session` command except the following:  
+* `START_DATE` should be the date of the first session
+* `END_DATE` should be the date of the last session
+* `INTERVAL` is the number of days to the next session <br/>
   e.g. `INTERVAL` = 7 for weekly sessions
-* `ENDDATE` should be an ending date of the recurring session, consistent with `DATE` and `INTERVAL`
 
-Examples:
-* `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80 b/7 e/2021-01-29`
+Example:
+* `add_rec_session n/John Doe d/2021-01-01 e/2021-01-29 b/7 t/18:00 k/120 s/Biology f/80` adds a weekly session that starts on 
+2021-01-01 and ends on 2021-01-29
+  
 #### Deleting a tuition session: `delete_session`
 
-Deletes the specified tuition session from TutorBuddy
+Deletes the specified tuition session from TutorBuddy.
 
 Format: `delete_session n/STUDENT_NAME i/SESSION_INDEX`
 
+* `STUDENT_NAME` should match the exact student’s name in TutorBuddy
 * Deletes the tuition session at the specified `SESSION_INDEX`
-* `SESSION_INDEX` refers to the session index respective to the student specified under `STUDENT_NAME`
+* `SESSION_INDEX` refers to the session index for a particular student
 * The index must be a positive integer 1, 2, 3, …​
```

Comment:
Might be better to flip points 2 and 3, so that it looks more 'uniform'

Reply:
True, also this could possibly make use of the ℹ️ notation we have included.